calculator.js

```diff
@@ -15,7 +15,12 @@ const divide = ((num1, num2) => {
     return num1 / num2;
 });
 
-// OPERATE FUNCION WITH SWITCH STATEMENT
+// OPERATE FUNCTION WITH SWITCH STATEMENT
+/* 
+Wow! Eric tells me this is a great way to write this function. 
```

Comment:
I never told you that. I told you this was the first transition toward something cool. Quit bad-mouthing me. My reputation is at stake.

Reply:
Calm down.